synapsis/src/main/java/com/soen/synapsis/appuser/jobposting/JobPosting.java

```diff
@@ -0,0 +1,90 @@
+package com.soen.synapsis.appuser.jobposting;
+
+import com.soen.synapsis.appuser.AppUser;
+import com.soen.synapsis.appuser.Role;
+
+public class JobPosting {
+
+    private final AppUser creator;
+    private String position;
+    private String company;
```

Comment:
This as well I think should be an ID foreign key to the company (and don't forget that companies are just appUsers with a company role).